lisa/tools/lscpu.py

```diff
@@ -55,7 +55,7 @@ def __repr__(self) -> str:
 class Lscpu(Tool):
     # CPU(s):              16
     # Total CPU(s):            2
```

Comment:
It's better to add negative cases and clearly explain it.

Reply:
addressed in #2264